src/syntax/unicode-tables.js

```diff
@@ -39,13 +39,14 @@ export var idStartTable = [
   886, 887,
   890, 890,
   891, 893,
+  895, 895,
```

Comment:
I guess the data at http://www.unicode.org/Public/UNIDATA/DerivedCoreProperties.txt got updated.